docs/reference/api/docker_remote_api.md

```diff
@@ -12,12 +12,11 @@ parent = "smn_remoteapi"
 
  - By default the Docker daemon listens on `unix:///var/run/docker.sock`
    and the client must have `root` access to interact with the daemon.
- - If the Docker daemon is set to use an encrypted TCP socket (`--tls`,
-   or `--tlsverify`) as with Boot2Docker 1.3.0, then you need to add extra
+ - If you are using `docker-machine`, the Docker daemon is on a virtual host that uses an encrypted TCP socket. In this situation, you need to add extra
```

Comment:
At this point, if they are using the API they are sophisticated enough to get the IP